requirements.txt

```diff
@@ -2,8 +2,8 @@ certifi==2018.11.29
 chardet==3.0.4
 docopt==0.6.2
 idna==2.8
-numpy==1.15.4
-pandas==0.23.4
+numpy==1.19.1
+pandas==1.1.1
```

Comment:
Looks good, could you please also make the same change in `requirements-dev.txt`?